picard/releasegroup.py

```diff
@@ -86,8 +90,8 @@ def _parse_versions(self, document):
             release = {
                 "id":      node['id'],
                 "year":    node['date'][:4] if "date" in node else "????",
-                "country": "+".join(countries) if countries
-                           else node.get('country', '') or "??",
+                "country": limited_join(countries, 10, '+', '…') if countries
+                else node.get('country', '') or "??",
                 "format":  media_formats_from_node(node['media']),
                 "label":  ", ".join([' '.join(x.split(' ')[:2]) for x in set(labels)]),
                 "catnum": ", ".join(set(catnums)),
```

Comment:
Should we handle limiting label and catnum as well?

Reply:
Yes, very likely, but that's not this ticket.